<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Test</class>
 <widget class="QDialog" name="Test">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="questionLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>711</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
    background-color: #4B0082; 
    color: white;            
    border-radius: 10px;        
    padding: 8px 12px;      
    font-weight: bold;     
    border: 2px solid #9370DB; 
}
qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255))</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="textFormat">
    <enum>Qt::TextFormat::MarkdownText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::ClickFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
    border: 2px solid rgb(255, 255, 255); 
    background: r #9370DB;        
    border-radius: 7px;      
    height: 12px;            
    text-align: center;        
}

QProgressBar::chunk {
    background-color: rgb(226, 83, 255);  
    width: 3px;                /* Очень узкие сегменты */
    border-radius: 5px;        /* Закругление сегментов */
    margin: 0.3px;             /* Минимальный отступ */
}</string>
   </property>
   <property name="maximum">
    <number>15</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="format">
    <string>%v/30</string>
   </property>
  </widget>
  <widget class="QPushButton" name="answerButton1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    /* Исходный градиент: от тёмно-фиолетового (#4B0082) к пурпурному (#8B008B) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #4B0082, stop:1 #8B008B);
    color: white;
    border-radius: 20px;
    border: none;
    padding: 10px;
}

QPushButton:hover {
    /* Новый градиент: добавляем ярко-розовый (#FF00FF) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #FF00FF, stop:1 #8B008B);
}

QPushButton:pressed {
    /* При нажатии — более тёмный розовый (#CC00CC) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #CC00CC, stop:1 #6A006A);
}</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="answerButton2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    /* Исходный градиент: от тёмно-фиолетового (#4B0082) к пурпурному (#8B008B) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #4B0082, stop:1 #8B008B);
    color: white;
    border-radius: 20px;
    border: none;
    padding: 10px;
}

QPushButton:hover {
    /* Новый градиент: добавляем ярко-розовый (#FF00FF) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #FF00FF, stop:1 #8B008B);
}

QPushButton:pressed {
    /* При нажатии — более тёмный розовый (#CC00CC) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #CC00CC, stop:1 #6A006A);
}</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="answerButton3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    /* Исходный градиент: от тёмно-фиолетового (#4B0082) к пурпурному (#8B008B) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #4B0082, stop:1 #8B008B);
    color: white;
    border-radius: 20px;
    border: none;
    padding: 10px;
}

QPushButton:hover {
    /* Новый градиент: добавляем ярко-розовый (#FF00FF) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #FF00FF, stop:1 #8B008B);
}

QPushButton:pressed {
    /* При нажатии — более тёмный розовый (#CC00CC) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #CC00CC, stop:1 #6A006A);
}</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="answerButton4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>370</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    /* Исходный градиент: от тёмно-фиолетового (#4B0082) к пурпурному (#8B008B) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #4B0082, stop:1 #8B008B);
    color: white;
    border-radius: 20px;
    border: none;
    padding: 10px;
}

QPushButton:hover {
    /* Новый градиент: добавляем ярко-розовый (#FF00FF) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #FF00FF, stop:1 #8B008B);
}

QPushButton:pressed {
    /* При нажатии — более тёмный розовый (#CC00CC) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #CC00CC, stop:1 #6A006A);
}</string>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="FinLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>711</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>25</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Опрос пройден</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>398</y>
     <width>271</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    /* Исходный градиент: от тёмно-фиолетового (#4B0082) к пурпурному (#8B008B) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #4B0082, stop:1 #8B008B);
    color: white;
    border-radius: 20px;
    border: none;
    padding: 10px;
}

QPushButton:hover {
    /* Новый градиент: добавляем ярко-розовый (#FF00FF) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #FF00FF, stop:1 #8B008B);
}

QPushButton:pressed {
    /* При нажатии — более тёмный розовый (#CC00CC) */
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #CC00CC, stop:1 #6A006A);
}</string>
   </property>
   <property name="text">
    <string>Завершить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
